src/ui/routes/apis.js

```diff
@@ -311,7 +323,60 @@ router.get('/:api', function (req, res, next) {
             apiInfo.hasProtectedAuthMethods = hasProtectedMethods;
             apiInfo.hasSwaggerApplication = hasSwaggerApplication;
             // See also views/models/api.json for how this looks
-            if (!utils.acceptJson(req)) {
+            const customId = JSON.stringify(userInfo.customId)
+            const trueid = customId.split(":")
+            const test1Url = trueid.length > 1 ? trueid[1] : null;
+            if (apiInfo.id === 'cortellies-api-collection'){
+                let responseData;
+                async.parallel({
+                    getTruid: (callback) => {
+                      const apiUrl = `https://api.dev-snapshot.clarivate.com/clarivate/entitlements/${test1Url}`;
+                      const headers = {
+                        'Content-Type': 'application/json',
+                        'X-ApiKey': '434654dee1cfa135a279b5b1220f2d3bf18d20c5'
```

Comment:
configure this key in ssm , don't hardcode

Reply:
okay will use 
for  apiUrl ? should I have to use environment variables.